### ISSUE TITLE
What is the results of XLSR-53 model?

### ISSUE THREAD

**Issue by iamxiaoyubei** [pytorch/fairseq#2997]
Thank you for your valuable sharing. Recently, I saw that you published the XLSR-53 model. What is the result of it on babel, common voice, and MLS?

**Comment by iamxiaoyubei** [comment]
Besides, can you share the config for running XLSR-53 (finetuning and pretraining)?

**Comment by aconneau** [comment]
@iamxiaoyubei Hey sorry I missed this. You can find the results of the XLSR-53 model on MLS, CommonVoice and BABEL in our update XLSR paper here: https://arxiv.org/abs/2006.13979

The pretraining is similar to the pretraining of the Librivox model (same setting, with lower learning rate). Except the dataset is preprocessed in such a way that it oversamples lower-resource languages, just like in XLM, and as described in the XLSR paper.

For finetuning, the config are the same than for wav2vec 2.0 Librivox model: so you can find scripts for finetuning on 1 hour, 10 hours, 100 hours and 960 hours. Depending on the size of your dataset, you can choose to pick the right finetuning script.

**Comment by aconneau** [comment]
For pretraining, the code remains the same, the only thing that changes is in the ".tsv" file that you need to prepare.
And basically, in there, you simply add the path to the .wav (for instance) files of all your languages. So you have a single .tsv file for the 53 languages. But you "upsample" the lower-resource ones, by having multiple times low-resource language files for instance. This is a bit hacky but that means you don't have to make any change in the code, you simply have to modify the preprocessing.

For fine-tuning, you can download the MLS data and CommonVoice data online (BABEL you have to sign something to get it). Then you can do the preprocessing exactly how it's done for English: each character is space-separated and words are separated by a "|" (that creates your output .wrd file for instance).

For training language models for all those languages, I would suggest this data we recently released called "CC100": http://data.statmt.org/cc-100/. Not perfect because you need to filter it to fit the domain of (e.g.) spoken language, but still a good resource.

**Comment by iamxiaoyubei** [comment]
@aconneau Thank you~!
